frontend/src/features/dsPreview/state/dsPreviewActions.ts

```diff
@@ -12,7 +12,7 @@ function fetchDsPreview (ref: QriRef): ApiAction {
     type: 'datasetpreview',
     ref,
     [CALL_API]: {
-      endpoint: 'dataset_summary',
+      endpoint: 'ds/get',
```

Comment:
cloud & local differ here. In my code spike I've adjusted `qri get` with no selector to return a preview

Reply:
q: Should this be the case? Shouldn't get return the "full" response while previews should be separate. We do also have `/remote/previews` albeit a bit different thing there. I guess using `/ds/get` to fetch previews should be fine once cursors and proper paging are set up. 

Still not sure if we should bend `get` into this. And am more for setting up `/ds/preview` for it for now. The route fix as requested can land as soon as the changes in core for `get` are on master.